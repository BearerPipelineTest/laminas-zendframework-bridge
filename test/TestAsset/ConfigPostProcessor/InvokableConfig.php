<?php
return [
    'dependencies' => [
        'invokables' => [
            'console-helper'                                   => 'Zend\Stdlib\ConsoleHelper',
            'Zend\Stdlib\ConsoleHelper'                        => 'Zend\Stdlib\ConsoleHelper',
            'Zend\Stdlib\StringWrapper\StringWrapperInterface' => 'Zend\Stdlib\StringWrapper\Intl',
        ],
    ],
];
