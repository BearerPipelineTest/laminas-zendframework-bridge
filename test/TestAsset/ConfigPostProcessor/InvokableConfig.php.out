<?php
return [
    'dependencies' => [
        'invokables' => [
            'console-helper' => 'Laminas\Stdlib\ConsoleHelper',
            'Zend\Stdlib\ConsoleHelper' => 'Laminas\Stdlib\ConsoleHelper',
            'Zend\Stdlib\StringWrapper\StringWrapperInterface' => 'Laminas\Stdlib\StringWrapper\Intl',
            'Laminas\Stdlib\ConsoleHelper' => 'Laminas\Stdlib\ConsoleHelper',
            'Laminas\Stdlib\StringWrapper\StringWrapperInterface' => 'Laminas\Stdlib\StringWrapper\Intl',
        ],
    ],
];
